libs/build-all.sh

```diff
@@ -54,7 +54,13 @@ if [[ ! -e "${SQLCIPHER}.tar.gz" ]]; then
 else
  echo "Using ${SQLCIPHER}.tar.gz"
 fi
-echo "${SQLCIPHER_SHA256}  ${SQLCIPHER}.tar.gz" | shasum -a 256 -c - || exit 2
+# Integrity check for SQLCIPHER
+if ! echo "${SQLCIPHER_SHA256}  ${SQLCIPHER}.tar.gz" | shasum -a 256 -c - 
+then
+    echo "Error: ${SQLCIPHER}.tar.gz is corrupted. Please try running this build script again."
```

Comment:
A very small suggestion for clarity in the error message - since we automatically deleted the file, it's more accurate to say that it *was* corrupted (but we've fixed it, so the user can safely try again) rather than to say it *is* corrupted (which might cause the user to think they have to fix it themselves somehow):

```suggestion
    echo "Error: ${SQLCIPHER}.tar.gz was corrupted. Please try running this build script again."
```